gradle/test-libs.versions.toml

```diff
@@ -4,6 +4,7 @@ allure = "2.29.1"
 assertj = "3.27.3"
 testcontainers = "1.21.3"
 aspectj = "1.9.24"
+nginxContainer = "1.21.3"
```

Comment:
Testcontainers releases use the same version number for all modules, so you can reuse the already defined version `testcontainers` also for the new dependency.

Reply:
Done